src/NUnitEngine/nunit.engine/Internal/RuntimeFrameworks/DotNetFrameworkLocator.cs

```diff
@@ -1,5 +1,5 @@
 // ***********************************************************************
-// Copyright (c) 2007 Charlie Poole, Rob Prouse
+// Copyright (c) 2018 Charlie Poole, Rob Prouse
```

Comment:
Any particular reason for this change? If the thinking was this being a 'new' file - I originally kept the 2007 date as I'd only moved code here, and not written anything new.

I don't know the specifics on what we _should_ be doing in this case, however! :smile: